tests/servers.go

```diff
@@ -0,0 +1,40 @@
+package tests
```

Comment:
This goes to `server.go` I think

Reply:
I have yet to come up with a decent proposal for that ...

In my head, they're not the same; I the stuff you've provided me as a set of utilities that'll give me a functionality running isolated in a pod. 

This is just an adaptation of the code we have to start servers on target running VMIs.

I agree that it cannot be named `servers` and `server`. But I think they don't belong together. 

Any suggestion about the naming ? 

A different alternative would be to extend the https://github.com/kubevirt/kubevirt/blob/master/tests/job.go#L142 having a 'server side' http / tcp job.

Reply:
proper thing is libserver/vmi.go and libserver/pod.go or porserver.go and vmiserver.go